.docker-compose-okteto.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+services:
+  mongo:
+    image: "mongo:latest"
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=root
+      - MONGO_INITDB_ROOT_PASSWORD=password
+  growthbook:
+    build: .
+    ports:
+      - "3000:3000"
+      - "3100:3100"
+    depends_on:
+      - mongo
+    environment:
+      - MONGODB_URI=mongodb://root:password@mongo:27017/
+      - APP_ORIGIN=https://growthbook-3000-${OKTETO_NAMESPACE}.cloud.okteto.net
+      - API_HOST=https://growthbook-3100-${OKTETO_NAMESPACE}.cloud.okteto.net
+      - JWT_SECRET=${JWT_SECRET}
+      - IS_CLOUD=true
```

Comment:
wondering if there's a way to configure the env or specifically IS_CLOUD based on build parameters?

Reply:
What I meant is letting the PR author (user) pass in some sort of env variable into the job / docker-compose file. For example, we could manually override the IS_CLOUD={true|false} or other properties. Something like this...
https://blog.geralexgr.com/github/parameters-and-variables-github-workflows

Anyhow don't let this block you, just an idea for the future.

Reply:
Got it. Makes sense, I did not know github allowed that for its workflows ... In any case, these defaults we've selected should work for the majority of cases ... Good to know we have this option for the future if we need it